[2025-07-02 09:31:11 UTC] Starting Spark Job with 1 partition(s).
[2025-07-02 09:31:16 UTC] Spark created 5 task(s): 0 ACTIVE, 5 COMPLETED, 0 FAILED tasks
[2025-07-02 09:31:16 UTC] Processed all records
[2025-07-02 09:31:16 UTC] ********** ERRORS **********
[2025-07-02 09:31:16 UTC] [ERROR] 1 record(s) failed with KeyError: 'GOOGLE_API_TOKEN'
[2025-07-02 09:31:16 UTC] ---------------------------------------------------------------------------
[2025-07-02 09:31:16 UTC] KeyError                                  Traceback (most recent call last):
[2025-07-02 09:31:16 UTC]  FoundationModel "Gemini 2.5 Flash" <v1>, line 15, in <definition>
[2025-07-02 09:31:16 UTC]         12 | if "client" in cache:
[2025-07-02 09:31:16 UTC]         13 |     client = cache["client"]
[2025-07-02 09:31:16 UTC]         14 | else:
[2025-07-02 09:31:16 UTC] ---->   15 |     client = init()
[2025-07-02 09:31:16 UTC]                           ^^^^^^
[2025-07-02 09:31:16 UTC]         16 |     cache["client"] = client
[2025-07-02 09:31:16 UTC]     Local variables:
[2025-07-02 09:31:16 UTC]       text          = 'qsqssq'
[2025-07-02 09:31:16 UTC]       temperature   = 1.0
[2025-07-02 09:31:16 UTC]       cache         = Cache({}, maxsize=1024, currsize=0)
[2025-07-02 09:31:16 UTC]  FoundationModel "Gemini 2.5 Flash" <v1>, line 8, in init
[2025-07-02 09:31:16 UTC]          6 | def init():
[2025-07-02 09:31:16 UTC]          7 |     # return genai.Client(api_key="AIzaSyDgkWZDDfEjWKMUpJ0R5yV8z7yvONLKN50")
[2025-07-02 09:31:16 UTC] ---->    8 |     print("##############", os.environ["GOOGLE_API_TOKEN"])
[2025-07-02 09:31:16 UTC]                                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[2025-07-02 09:31:16 UTC]          9 |     return genai.Client(api_key=os.getenv("GOOGLE_API_TOKEN"))
[2025-07-02 09:31:16 UTC]  FoundationModel "Gemini 2.5 Flash" <v1>, line 428, in __getitem__
[2025-07-02 09:31:16 UTC]     Local variables:
[2025-07-02 09:31:16 UTC]       self          = environ({'PATH': '/usr/...LIFIED_TRACEBACK': '1'})
[2025-07-02 09:31:16 UTC]       key           = 'GOOGLE_API_TOKEN'
[2025-07-02 09:31:16 UTC] KeyError: 'GOOGLE_API_TOKEN'
[2025-07-02 09:31:23 UTC] Execution completed - 0 records ran successfully and 1 records had errors
[2025-07-02 09:31:23 UTC] Got 0 records after execution. Unable to find any data to run reports on.
